database/seeders/AccountSeeder.php

```diff
@@ -5,10 +5,21 @@
 use Entities\Models\Eloquent\Account;
 use Entities\Models\Eloquent\Group;
 use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Crypt;
 use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+use PragmaRX\Google2FA\Google2FA;
 
 class AccountSeeder extends Seeder
 {
+    private $google2FA;
+
+    public function __construct(Google2FA $google2FA)
+    {
+        $this->google2FA = $google2FA;
+    }
```

Comment:
PHP 8 lets us skip this boilerplate code now 😄 

```suggestion
    public function __construct(
        private Google2FA $google2FA,
    ) {}
```